packages/core/admin/admin/src/translations/en.json

```diff
@@ -34,6 +37,7 @@
   "Auth.form.lastname.label": "Lastname",
   "Auth.form.lastname.placeholder": "Doe",
   "Auth.form.password.label": "Password",
+  "Auth.form.password.hint": "Password must contain at least 8 characters, 1 uppercase, 1 lowercase and 1 number",
```

Comment:
I think this comment is wrong, although I'm looking for the regex pattern we use. I think a symbol is required as well.

Reply:
Currently, the symbol isn't required for the admin registration form.